plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -1291,4 +1293,17 @@ public HiveConfig setPartitionProjectionEnabled(boolean enabledAthenaPartitionPr
         this.partitionProjectionEnabled = enabledAthenaPartitionProjection;
         return this;
     }
+
+    public boolean isLegacyAcidModificationEnabled()
+    {
+        return isLegacyAcidModificationEnabled;
+    }
+
+    @Config("hive.legacy-acid-modification-enabled")
+    @ConfigDescription("Enable performing legacy DELETE/UPDATE on Hive ACID transactional tables")
```

Comment:
We are disabling it because it has correctness issues.
Why would we have a toggle for enabling it at all?

if we keep it, the description should scream loud that you should never use it.